main.go

```diff
@@ -163,20 +164,21 @@ func main() {
 			os.Exit(1)
 		}
 
-		tokenClaims, err := newKubeTokenClaimsFromFile(*flagAWSKubeTokenPath)
-		if err != nil {
-			log.Error(err, "error reading token from file", "file", *flagAWSKubeTokenPath)
+		pns := os.Getenv("POD_NAMESPACE")
+		psa := os.Getenv("POD_SERVICE_ACCOUNT")
+		if pns == "" || psa == "" {
+			log.Error(errors.New("POD_NAMESPACE and POD_SERVICE_ACCOUNT need to be set"), "")
 			os.Exit(1)
 		}
 
 		kubeAuthRole := *flagAWSKubeAuthRole
 		if kubeAuthRole == "" {
-			kubeAuthRole = *flagAWSPrefix + "_aws_" + tokenClaims.Namespace + "_" + tokenClaims.ServiceAccountName
+			kubeAuthRole = *flagAWSPrefix + "_aws_" + pns + "_" + psa
 		}
 
 		awsRole := *flagAWSRole
 		if awsRole == "" {
-			awsRole = *flagAWSPrefix + "_aws_" + tokenClaims.Namespace + "_" + tokenClaims.ServiceAccountName
+			awsRole = *flagAWSPrefix + "_aws_" + pns + "_" + psa
 		}
```

Comment:
`POD_NAMESPACE` and `POD_SERVICE_ACCOUNT` are required, even if you're explicitly setting the roles with flags and not using the environment variable values at all. 

Perhaps we should remove the flags and just hardcode the expected role name structure.

Or make the flags required and use the environment variables that way:
```
        - name: aws-credentials
          image: quay.io/utilitywarehouse/vault-kube-cloud-credentials:0.6.3
          args:
            - aws-sidecar
            - -role=vkcc_aws_$(POD_NAMESPACE)_$(POD_SERVICE_ACCOUNT)
            - -kube-auth-role=vkcc_aws_$(POD_NAMESPACE)_$(POD_SERVICE_ACCOUNT)
```

Reply:
I agree on them being required, thats why line 169 - makes sure there are values. Am I misunderstanding you?

Reply:
If you set `-role` (`*flagAWSRole`) and `-kube-auth-role` (`*flagAWSKubeAuthRole`) then there's no need for the environment variables, but they're still required.